packages/feature-toggles-client/src/index.ts

```diff
@@ -0,0 +1,34 @@
+export interface FeatureValue {
+    raw: any
+    string(): string
+    boolean(): boolean
+}
+
+/**
+ * Current state of the feature toggles
+ */
+export interface FeatureState {
+    [feature: string]: {
+        value: FeatureValue
+
+        /** User overridable */
+        canOverride: boolean
```

Comment:
Would calling it canUserOverride make things slightly clearer thus negating the need for the comment?

Reply:
Good suggestion